xbmc/addons/kodi-addon-dev-kit/include/kodi/AddonBase.h

```diff
@@ -82,7 +82,9 @@ typedef enum ADDON_STATUS
   ///
   ADDON_STATUS_UNKNOWN,
 
+  /// @warning obsolete and no more used from Kodi!
   ///
+  /// Use kodi::SetSetting... during end to store changed settings
   ADDON_STATUS_NEED_SAVEDSETTINGS,
```

Comment:
No, was only used from Visualization addons where already changed.
The old way: https://github.com/AlwinEsch/visualization.shadertoy/blob/c80118548589826f43938de0bb8be25cbf9ed728/src/maindx.cpp#L1110-L1142

Can be everything done with e.g. "kodi::SetSettingInt("lastpresetidx", lastIndex);"